doc/crds/daemonset-install.yaml

```diff
@@ -130,10 +143,16 @@ spec:
           mountPath: /host/opt/cni/bin
         - name: cni-net-dir
           mountPath: /host/etc/cni/net.d
+        - name: cron-scheduler-configmap
+            mountPath: /cron-schedule
```

Comment:
wrong indentation. I guess this is why the daemonset does not start in e2e tests